publication-commons/src/main/java/no/unit/nva/publication/rightsretention/RightsRetentionsValueFinder.java

```diff
@@ -54,7 +54,7 @@ public RightsRetentionStrategy getRightsRetentionStrategy(File file, Publication
 
     private boolean rrsIsIrrelevant(File file, Publication publication) {
         return !PublisherVersion.ACCEPTED_VERSION.equals(file.getPublisherVersion())
-               || file instanceof AdministrativeAgreement
+               || file instanceof InternalFile
                || !isAcademicArticle(publication);
     }
```

Comment:
One of few changes in business logic, someone has to verify that it is correct change.

Reply:
HiddenFile should also probably be here?